tests/cli/test_validate_json_telemetry_and_dry_run.py

```
@@ -0,0 +1,104 @@
+
+def test_json_counts_with_telemetry(tmp_path: Path) -> None:
+    yml = (
+        'version: "0.1"\n'
+        "steps:\n"
+        '  - name: A\n    agent: { id: "flujo.builtins.stringify" }\n    input: "{{ previous_step.output }}"\n'
+    )
+    f = tmp_path / "p.yaml"
+    f.write_text(yml)
+    env = os_environ_copy()
+    env["FLUJO_CLI_TELEMETRY"] = "1"
+    res = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)